src/lightning_app/core/flow.py

```diff
@@ -180,6 +183,8 @@ def __setattr__(self, name, value):
             elif _is_json_serializable(value):
                 self._state.add(name)
 
+                # Question: Just wondering, should we even allow users to even change the values to
```

Comment:
In theory yes, we would need to create a BaseType users can rely upon to create special behaviours with the state like Payload for example.